tests/e2e/specs/IntermediateCatchEvent.spec.js

```diff
@@ -42,14 +42,11 @@ describe('Intermediate Catch Event', () => {
     const nameInput = '[name=name]';
     const testString = 'testing';
     typeIntoTextInput(nameInput, testString);
-
     cy.contains('Timing Control').click();
     cy.get('[data-test=intermediateTypeSelect]').select('Date/Time');
-    const startDateValue = '2019-02-27';
-    typeIntoTextInput('.start-date', startDateValue);
-
-    const startTimeValue = '00:30';
-    cy.get('[data-test=startTime]').select(startTimeValue, { force: true });
+    const startDateTime = '02/27/2019 12:30 AM';
```

Comment:
When I looked at this test, I tried to click the time in the new date/time picker. It just didn't seem easy to do. So I'm ok with this.